napalm_ios/ios.py

```diff
@@ -670,15 +675,61 @@ def get_optics(self):
 
         return optics_detail
 
+    def get_cdp_neighbors(self):
```

Comment:
I would agree were it not that it might seem awkward for people running CDP in their network to have to make a call to `get_lldp_neighbors()`.